packages/react-accordion/src/components/AccordionPanel/useAccordionPanel.ts

```diff
@@ -16,15 +16,17 @@ export const accordionPanelShorthandProps: Array<keyof AccordionPanelSlots> = []
  */
 export const useAccordionPanel = (props: AccordionPanelProps, ref: React.Ref<HTMLElement>): AccordionPanelState => {
   const { open } = useAccordionItemContext();
-  const id = useId('accordion-panel-', props.id);
```

Comment:
Is it okay that we lost this? Should it be on `root`?

Reply:
This has nothing to do with the root as slot changes, I'm sorry. I'll try to rollback this!